src/components/item/sharing/SharingLink.tsx

```diff
@@ -88,10 +85,20 @@ const SharingLink: FC<Props> = ({ itemId }) => {
     if (link) {
       copyToClipboard(link, {
         onSuccess: () => {
-          notifier({ type: COPY_ITEM_LINK_TO_CLIPBOARD.SUCCESS, payload: {} });
+          notifier({
+            type: COPY_ITEM_LINK_TO_CLIPBOARD.SUCCESS,
+            payload: { message: translateBuilder('Link copied to clipboard') },
+          });
         },
         onError: () => {
-          notifier({ type: COPY_ITEM_LINK_TO_CLIPBOARD.FAILURE, payload: {} });
+          notifier({
+            type: COPY_ITEM_LINK_TO_CLIPBOARD.FAILURE,
+            payload: {
+              message: translateBuilder(
+                'There was an error copying the link to the clipboard',
```

Comment:
Should we add these strings in translations?

Reply:
@pyphilia see https://github.com/graasp/graasp-translations/pull/110